prime-dotnet-webapi/Models/Api/EnrolleeCertDto.cs

```diff
@@ -8,6 +8,7 @@ public class EnrolleeCertDto
         public string PractRefId { get; set; }
         public string CollegeLicenceNumber { get; set; }
         public string PharmaNetId { get; set; }
+        public string CollegeName { get; set; }
```

Comment:
I think this isn't necessary as the front-end has the data to do the translation from `CollegeCode`, e.g. see `CertificationViewModel`

Reply:
I wish I noticed that before. Code is updated accordingly.